control/control.go

```diff
@@ -104,3 +92,26 @@ func (s *HTTPServer) getEnvHandler(w http.ResponseWriter, r *http.Request) {
 	log.Debugf("marshaled environ: %v", string(envJSON))
 	w.Write(envJSON)
 }
+
+// postReloadHandler reloads ContainerPilot process
+func (s *HTTPServer) postReloadHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		failedStatus := http.StatusNotImplemented
+		log.Errorf("'POST %v' not responding to request method '%v'", r.URL, r.Method)
```

Comment:
The wording of this error message with "POST in front of the URL" implies that we got a POST but doesn't respond to some other method. I feel like this will be a little confusing when the problem might be that the user expected they were sending POST and didn't. We can be really explicit and simple here: `log.Errorf("%s requires POST, not %s", r.URL, r.Method)`